.travis.yml

```diff
@@ -35,8 +35,9 @@ env:
     - MODE=e2e
     - MODE=saucelabs_required
     - MODE=browserstack_required
-    - MODE=saucelabs_optional
-    - MODE=browserstack_optional
+# Temporary disable the optional builds because of potential concurrency flakiness
+#    - MODE=saucelabs_optional
+#    - MODE=browserstack_optional
```

Comment:
Rather than commenting it out, just remove them completely. Maybe someday we'll add them back.